fiftyone/utils/rfdetr.py

```
@@ -0,0 +1,411 @@
+    Args:
+        model_type: the RF-DETR class name, e.g. ``"RFDETRMedium"``
+        device (None): an optional device string to forward upstream
+
+    Returns:
+        an :class:`rfdetr.detr.RFDETR` instance
+    """
+    model_cls = getattr(rfdetr_lib, model_type, None)
+    if model_cls is None:
+        raise ValueError(
+            "Unknown RF-DETR model type '%s'. Available types: %s"
+            % (model_type, ", ".join(_VALID_MODEL_TYPES))
```

Ruff (0.15.14)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)